packages/xo-server-sdn-controller/src/ovsdb-client.js

```diff
@@ -11,6 +11,24 @@ const OVSDB_PORT = 6640
 // =============================================================================
 
 export class OvsdbClient {
+  /*
+  Create an SSL connection to an XCP-ng host.
+  Interact with the host's OpenVSwitch (OVS) daemon to create and manage the virtual bridges
+  corresponding to the private networks with OVSDB (OpenVSwitch DataBase) Protocol.
+  See:
+  - OVSDB Protocol: https://tools.ietf.org/html/rfc7047
+  - OVS Tunneling : http://docs.openvswitch.org/en/latest/howto/tunneling/
+
+  Attributes on created OVS bridges and ports
+  - `other_config`:
+    - `xo:sdn-controller:cross-pool`       : UUID of the remote network connected by the tunnel
+    - `xo:sdn-controller:private-pool-wide`: `true` if created (and managed) by a SDN Controller
+
+  - `options`:
```

Comment:
On which objects are created these entries?

Reply:
OVS interfaces. I'll add it in the comment.